### ISSUE TITLE
Seriously?

### ISSUE THREAD

**Issue by LIZI-Neazle** [csharpfritz/Fritz.Junity2020Pinball#19]
![GIF 2022-1-13 4-14-16](https://user-images.githubusercontent.com/47855670/149215397-dc1f5180-97e7-4436-a133-51a3df3aebb3.gif)

**Comment by csharpfritz** [comment]
Yep... it doesn't work, and I haven't gone back to investigate and fix things.

I would consider this project paused for now
